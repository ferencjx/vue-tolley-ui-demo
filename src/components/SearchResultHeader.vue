<template>
    <div class="searchResultCount">{{resultsCount}} results</div>
</template>

<script>
export default {
  name: 'SearchResultHeader',
  props: {
    resultsCount: Number
  }
}
</script>

<style scoped>
.searchResultCount{
  text-align: left;
  padding: 10px;
  width: 146px;
  height: 12px;
  font-family: Lato;
  font-size: 10px;
  font-weight: normal;
  font-style: normal;
  font-stretch: normal;
  line-height: normal;
  letter-spacing: normal;
  color: #373739;
}
</style>
