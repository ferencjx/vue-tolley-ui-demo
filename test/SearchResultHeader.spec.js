import { shallowMount } from '@vue/test-utils'

import SearchResultHeader from '../src/components/SearchResultHeader'

describe('SearchResultHeader', () => {
    it('renders the page', () => {
      const wrapper = shallowMount(SearchResultHeader)
      expect(wrapper).toMatchSnapshot()
    })
  })