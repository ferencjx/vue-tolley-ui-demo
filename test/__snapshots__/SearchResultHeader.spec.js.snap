// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResultHeader renders the page 1`] = `<div class="searchResultCount">0 results</div>`;
